src/views/v2/month/calendar-body/day/multiday-events/date/featured-hidden.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * View: Month View - Calendar Event Featured Icon
```

Comment:
This should be multiday event

Reply:
also should be under `month/calendar-body/day/mutliday-events/multiday-event/date/featured-hidden`